app/services/git_based_domain_import_service.rb

```diff
@@ -62,6 +81,16 @@ def import(git_repo_id, branch_or_tag, tenant_id)
     domain.update_attribute(:enabled, true)
   end
 
+  def refresh(git_repo_id)
```

Comment:
Why are we making that decision here?  I would think it is better to pass the object around and allow the lower method to decide what properties from the object they need.  I would think we would want all the methods to work that way.

We made a similar change when passing around the user information.  To keep the user object's identify (user, group, tenant) intact it was much better to pass around the object.